### PR TITLE
Update local installation guide for macOS installation

### LINKED ISSUE #3618: Open event Orga server generic installation on Mac OSX
**I'm submitting a ...**  (check one with "x")
- [x] feature request

**Current behavior:**
Currently, there is no documentation for OSX.

**Expected behavior:**
It'll be nice to have OSX documentation as well.

### PR DESCRIPTION
**Pull request by party98**
Fixes #3618 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Updates the local installation guide to add general installation instructions for macOS users.

#### Changes proposed in this pull request:
Updated local installation instructions for macOS users
-
-
-

### CODE REVIEW

**Comment by party98** [comment]
@rafalkowalski Done

File: app/api/attendees.py [hunk 1]
```diff
@@ -31,6 +31,12 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
+        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
```
Comment (rafalkowalski) [inline app/api/attendees.py]: Can you assign int(data['ticket']) to variable?

Comment (rafalkowalski) [reply app/api/attendees.py]: And check if data['ticket'] is digit. 
if data['ticket'].isdigit():
    ....

File: app/api/tickets.py [hunk 2]
```diff
@@ -31,6 +33,9 @@ def before_post(self, args, kwargs, data):
             raise ObjectNotFound({'parameter': 'event_id'},
                                  "Event: {} not found".format(data['event_id']))
 
+        if get_count(db.session.query(Ticket.id).filter_by(name=data['name'], event_id=int(data['event']))) > 0:
```
Comment (rafalkowalski) [inline app/api/tickets.py]: You need to check if data['event'] is digit or use try except

File: app/api/attendees.py (hunk 1 above)
Comment (poush) [inline app/api/attendees.py]: Why any of these changes in this PR? Completely unrelated to the the issue and title.

**Comment by party98** [comment]
@rafalkowalski  Fixed the merging issue :/

**Pull request merged**